### PR TITLE
Adds support for retrieving run logs

### PR DESCRIPTION
**Pull request by merschformann**
# Description

Adds support for retrieving logs of a run.

## Changes

- Adds `run_logs` func to the application for retrieving logs of a run

### CODE REVIEW

**Review by sebastian-quintero — APPROVED** [review]
Non-blocking remark.

File: nextmv/cloud/application.py [hunk 1]
```diff
@@ -646,6 +653,28 @@ def run_input(
...
+            Logs of the run.
+
+        Raises:
+            requests.HTTPError: If the response status code is not 2xx.
+        """
+        response = self.client.request(
+            method="GET",
+            endpoint=f"{self.endpoint}/runs/{run_id}/logs",
+        )
+        return RunLog.from_dict(response.json())
```
Comment (sebastian-quintero) [inline nextmv/cloud/application.py:676]: I think the pattern we are following so far is to return the actual class. People can then do whatever they want with it, like converting it to a dict. Up to you.

**Review by sebastian-quintero — APPROVED** [review]
The comment didn’t come through. Minor remark.

File: nextmv/cloud/application.py (hunk 1 above)
Comment (merschformann) [reply nextmv/cloud/application.py:676]: Good catch! Fixed it. :hugs:

**Pull request merged**